aspnetcore/fundamentals/openapi/aspnetcore-openapi.md

```diff
@@ -346,6 +351,21 @@ app.MapGet("/attributes",
   () => "Hello world!");
 ```
 
+##### [Controllers](#tab/controllers)
+
+In controller-based apps, the <xref:Microsoft.AspNetCore.Mvc.ApiExplorerSettingsAttribute> attribute can be used to exclude an endpoint from the OpenAPI document.
+
+The following example demonstrates how to exclude an endpoint from the generated OpenAPI document.
```

Comment:
Do we want to add that this also applies for committing entire controller classes?